code/modules/mob/living/basic/lavaland/bileworm/bileworm_instrument.dm

```diff
@@ -0,0 +1,18 @@
+/obj/item/instrument/bilehorn
+	name = "bilehorn"
+	desc = "Bits of bileworm anatomy rearranged to produce wonderful music, not bile. Keeps the name though, because for an instrument, it is quite vile."
+	force = 5
+	icon = 'icons/mob/lavaland/bileworm.dmi'
+	icon_state = "bilehorn"
+	allowed_instrument_ids = "bilehorn"
+	inhand_icon_state = "bilehorn"
+
+/datum/crafting_recipe/bilehorn
+	name = "Bilehorn"
+	reqs = list(
+		/obj/item/stack/sheet/animalhide/bileworm = 4,
+		/obj/item/stack/sheet/bone = 2,
+		/obj/item/crusher_trophy/bileworm_spewlet = 1,
```

Comment:
this will make the bilehorn only accessible to crusher users, do we want that

Reply:
I'll make an alternate way to get one.